src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -52,17 +53,37 @@ static std::pair<size_t, size_t> get_output_aligned_bf_size(const fully_connecte
     return {output_b, output_f};
 }
 
+static size_t get_scale_group_size(const fully_connected_params& params) {
+    return params.weights.IFM().v / params.decompression_scale.Feature().v;
+}
+
+static bool is_dyn_quan_8bit_asym(const fully_connected_params& params) {
```

Comment:
* we need to check `asym` with zp, not the data type. JFYI, Sergey implemented asymmetric quantization with i8.
* What about using more clear name like `8bit_asym_wei`?